src/Tickets/Emails/Email/Ticket.php

```diff
@@ -199,43 +99,42 @@ public function get_default_heading_multiple() {
 	}
 
 	/**
-	 * Get email subject.
-	 *
-	 * @since 5.5.9
-	 *
-	 * @return string The email subject.
+	 * Get heading for plural tickets.
+	 * 
+	 * @since TBD
+	 * 
+	 * @return string
 	 */
-	public function get_subject(): string {
-		$subject = tribe_get_option( static::$option_subject, $this->get_default_subject() );
-
-		$subject = $this->format_string( $subject );
+	public function get_heading_plural(): string {
+		$option_key = $this->get_option_key( 'heading-plural' );
+		$heading = tribe_get_option( $option_key, $this->get_default_heading_plural() );
 
 		// @todo: Probably we want more data parsed, or maybe move the filters somewhere else as we're always gonna
-		// apply filters on the subject maybe move the filter to the parent::get_subject() ?
 
 		/**
-		 * Allow filtering the email subject.
+		 * Allow filtering the email heading globally.
 		 *
-		 * @since 5.5.9
+		 * @since TBD
 		 *
-		 * @param string          $subject  The email subject.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
+		 * @param string $heading  The email heading.
+		 * @param string $id       The email id.
+		 * @param string $template Template name.
 		 */
-		$subject = apply_filters( 'tec_tickets_emails_ticket_subject', $subject, self::$id, $this->template );
+		$heading = apply_filters( 'tec_tickets_emails_heading_plural', $heading, $this->id, $this->template );
 
 		/**
-		 * Allow filtering the email subject.
+		 * Allow filtering the email heading for Completed Order.
 		 *
-		 * @since 5.5.9
+		 * @since TBD
 		 *
-		 * @param string          $subject  The email subject.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
+		 * @param string $heading  The email heading.
+		 * @param string $id       The email id.
+		 * @param string $template Template name.
 		 */
-		$subject = apply_filters( 'tec_tickets_emails_subject', $subject, self::$id, $this->template );
+		$template_name = static::$template;
+		$heading = apply_filters( "tec_tickets_emails_{$template_name}_heading_plural", $heading, $this->id, $this->template );
```

Comment:
We should use the ID as part of the hook filter. The idea is that people will be able to extend this functionality and use their own templates if they want. The template is meant to store the filename that's going to be called to get the content of the email, and that _could_ change if extended. The ID will always be the same for the email.

Reply:
I created a new variable specifically for filters and options called `slug`. That way filter and option names won't be so long.